R/gather_ard.R

```diff
@@ -0,0 +1,41 @@
+#' Extract ARDs
+#'
+#' Extract the ARDs from a gtsummary table. If needed, results may be combined
+#' with `cards::bind_ard()`.
+#'
+#' @param x (`gtsummary`)\cr a gtsummary table
+#'
+#' @return list
+#' @export
+#'
+#' @examplesIf gtsummary:::is_pkg_installed('cardx', reference_pkg = 'gtsummary')
```

Comment:
shouldnt it be present anyhow?

Reply:
Nearly 100% of the time, yes. But there is a special type of R CMD Check that runs without using any of the Suggested packages, and we're meant to fail gracefully in these situations. So in this case, we just don't run the examples that require any suggested packages.